File: src/Java/CreationalPatterns/Builder/Laptops/src/Director/LaptopStore.java
package Director;

import Builder.LaptopBuilder;

public class LaptopStore {
    public void buildLaptopConfiguration(LaptopBuilder laptopBuilder) {
        laptopBuilder.setModel();
        laptopBuilder.setCPUSeries();
        laptopBuilder.setGPUModel();
        laptopBuilder.setRAMType();
        laptopBuilder.setRAMSize();
        laptopBuilder.setDisplayType();
        laptopBuilder.setSSDType();
        laptopBuilder.setSSDCapacity();
        laptopBuilder.setExtras();
    }
}

File: src/PHP/CreationalPatterns/Builder/Laptops/Director/LaptopStore.php
<?php

namespace CreationalPatterns\Builder\Laptops\Director;

use CreationalPatterns\Builder\Laptops\Builder\LaptopBuilder;

class LaptopStore
{
    public function buildLaptopConfiguration(LaptopBuilder $laptopBuilder): void
    {
        $laptopBuilder->setModel();
        $laptopBuilder->setCPUSeries();
        $laptopBuilder->setGPUModel();
        $laptopBuilder->setRAMType();
        $laptopBuilder->setRAMSize();
        $laptopBuilder->setDisplayType();
        $laptopBuilder->setSSDType();
        $laptopBuilder->setSSDCapacity();
        $laptopBuilder->setExtras();
    }
}